setup.js

```diff
@@ -91,4 +91,5 @@ let prompts = [
   console.log("REMEMBER TO NEVER SHARE YOUR TOKEN WITH ANYONE!");
   console.log("Configuration has been written, enjoy!");
   await settings.close();
-}());
\ No newline at end of file
+}());
+process.exit();
```

Comment:
Hold up. process.exit() outside the async function will make it exit before it even loads enmap. Please move this to line 94, and then it should be good!

Reply:
Oops, yes, my bad. Fixed it